### DISCUSSION TITLE
Active playbooks with no elastalert

### DISCUSSION THREAD

**Discussion by wildzealot** [Security-Onion-Solutions/securityonion#4279] [Unsupported Versions] [answered]
I am running security onion 2.3.5 as a standalone instance. I have activated 80 plays using Playbooks. When I check /opt/so/rules/elastalert/playbook/ there is only one .yml created. Shouldn't there be one for each active playbook?

**Comment by defensivedepth** [comment, marked as answer]
@wildzealot You can run so-playbook-sync to regenerate active Plays.
